x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/StringUtils.java

```diff
@@ -24,6 +23,8 @@
 import static java.time.temporal.ChronoField.SECOND_OF_MINUTE;
 
 public final class StringUtils {
+    public static final java.util.Locale WEEK_START_MON = new java.util.Locale.Builder()
```

Comment:
Why another copy? Can we use a single shared locale with this set to monday?

Reply:
this was here because the sql project is not depending on server

Reply:
Ok, then maybe do put it into it's own class, but in core/libs? It is all jdk based stuff so it should be fine there.